custom_components/adjustable_bed/beds/reverie_nightstand.py

```
@@ -0,0 +1,475 @@
+    async def stop_notify(self) -> None:
+        """Stop listening for position notifications."""
+        if self.client is None or not self.client.is_connected:
+            return
+
+        try:
+            await self.client.stop_notify(REVERIE_NIGHTSTAND_HEAD_POSITION_UUID)
+            await self.client.stop_notify(REVERIE_NIGHTSTAND_FOOT_POSITION_UUID)
+        except BleakError:
+            pass
+
+    async def read_positions(self, motor_count: int = 2) -> None:
```

Ruff (0.14.13)

Unused method argument: `motor_count`

(ARG002)